index.html

```diff
@@ -2183,67 +1880,272 @@ <h2>
+<a>DID document</a> to express an additional piece of information, the subject's
+public photo stream.
+        </p>
+
+        <p>
+The first thing that a developer would do is create a JSON-LD Context containing
+the new term.
+        </p>
+
+        <pre class="example nohighlight" title="An example JSON-LD Context">
+{
+  "@context": {
+    "PhotoStreamService": "https://example.com/vocab#PhotoStreamService"
+  }
+}
+        </pre>
+
+        <p>
+Now that the JSON-LD Context is created, the developer MUST publish it somewhere
+that is accessible to any <a>DID document</a> processor. For this example,
+assume that the JSON-LD Context above is published at
+<code>did:example:contexts:987654321</code>. At this point, extending the first
+example in this section is just a matter of including the context above and
+adding the new property to the <a>DID document</a>.
+        </p>
+
+        <pre class="example nohighlight" title="A DID document with a custom extension">
+{
+  "@context": "https://example.org/example-method/v1",
+  "id": "did:example:123456789abcdefghi",
+  "authentication": [ <span class="comment">...</span> ],
+  "service": [<span class="highlight">{
+    "@context": "did:example:contexts:987654321",
+    "id": "did:example:123456789abcdefghi#photos",
+    "type": "PhotoStreamService",
+    "serviceEndpoint": "https://example.org/photos/379283"
+  }</span>]
+}
+        </pre>
+
+        <p>
+The examples so far show that it is easy to extend the
+<a>decentralized identifiers</a> data model in a permissionless and
+decentralized way. The mechanism also ensures that
+<a>decentralized identifiers</a> created in this way prevent namespace conflicts
+and semantic ambiguity.
+        </p>
+
+        <p>
+An extensibility model this dynamic does increase the implementation burden.
+Software written for such a system needs to determine if accepting
+<a>DID documents</a> with extensions is acceptable based on the risk profile of
+the application. Some applications might choose to accept but ignore extensions,
+others might choose to only accept certain extensions, while highly secure
+environments might disallow extensions. These decisions are up to the
+application developers and are specifically not the domain of this
+specification.
+        </p>
+
+        <p>
+Implementations MUST produce an error when an extension JSON-LD Context
+overrides the expanded URL for a term specified in this specification. To avoid
+the possibility of accidentally overriding terms, developers SHOULD scope their
+extensions. For example, the following extension scopes the new
+<code>PhotoStreamService</code> term so that it can only be used within the
+<code>service</code> property:
+        </p>
+
+        <pre class="example nohighlight" title="Scoping terms in a JSON-LD Context">
+{
+  "@context": {
+    <span class="highlight">"service": {
+      "@id": "https://w3id.org/did#service",
```

Comment:
why are we referencing w3id here? can we not use the did core spec to define service?

Reply:
I find this example a little hard to grok... I would also prefer to see URIs that resolve to definitions.

Reply:
> why are we referencing w3id here? can we not use the did core spec to define service?

The example is wrong -- this was written 2+ years ago and it needs to be updated.